src/main/js/MergeModal.tsx

```diff
@@ -167,19 +193,28 @@ class MergeModal extends React.Component<Props, State> {
       </>
     );
 
-    const body = (
+    const body = (<>
       <MergeForm
         selectedStrategy={mergeStrategy}
         selectStrategy={this.selectStrategy}
         strategyLinks={pullRequest._links.merge as Link[]}
         commitMessage={mergeCommit.commitMessage}
         onChangeCommitMessage={this.onChangeCommitMessage}
-        onResetCommitMessage={this.getDefaultMessage}
+        onResetCommitMessage={this.getMergeStrategyInfo}
         shouldDeleteSourceBranch={mergeCommit.shouldDeleteSourceBranch}
         onChangeDeleteSourceBranch={this.onChangeDeleteSourceBranch}
         loading={loadingDefaultMessage}
+        commitMessageDisabled={commitStrategyInfos[mergeStrategy]?.commitMessageDisabled}
+        commitMessageHint={commitStrategyInfos[mergeStrategy]?.commitMessageHint}
       />
-    );
+      {mergeFailed && <>
+        <hr/>
+        <ErrorBanner
+          className="has-background-danger has-rounded-border">
+          {t('scm-review-plugin.showPullRequest.mergeModal.mergeConflict')}
+        </ErrorBanner>
```

Comment:
Shouldn't we use the well established `ErrorNotification` here?

Reply:
I think we should at least use the `Notification` component, yes. I didnt find it the first time i searched, although only briefly. I think `ErrorNotification` is more for actual exceptions as it is rather large. I would prefer a small banner component i think.